org.openhab.binding.zigbee/src/main/resources/discovery.txt

```diff
@@ -1,6 +1,7 @@
 philips_sml001,vendor=Philips,modelId=SML001
+philips_rwl012,vendor=Philips,modelId=RWL021
```

Comment:
Sorry, one last question, as I overlooked this before: Should it be `012` or `021` in the ID? It looks as if the digits were accidentally transposed on one side of this property entry.

Reply:
It should be rwl021. Thanks for the hint. I've fixed it.